etc/systemd/iscsi.service

```diff
@@ -2,9 +2,9 @@
 Description=Login and scanning of iSCSI devices
 Documentation=man:iscsiadm(8) man:iscsid(8)
 Before=remote-fs.target
-After=network.target network-online.target iscsid.service
-Requires=iscsid.socket
-ConditionPathExists=/etc/iscsi/initiatorname.iscsi
+After=network.target network-online.target
```

Comment:
Why did you remove the requirement to go after iscsid.service?

Reply:
I did not remove it, I just moved it one line below into the new After= statement to make lines shorter and have a nice symmetry with the Requires= statement another line below.